### DISCUSSION TITLE
Which sonarqube community version to select  10.6 or 9.9 LTA ?

### DISCUSSION THREAD

**Discussion by ccecvb** [Riverside-Software/sonar-openedge#1139] [Q&A] [answered]
We're upgrading our sonarqube community server.

I'm wondering wether it's best to stick with an LTA or to go to 10.6
https://www.sonarsource.com/blog/sonarqube-10-6-release-announcement/

**Comment by gquerret** [comment]
Stay with 9.9 if you still have to maintain 11.7 code with PDSOE.

**Reply by cverbiest** [reply]
@gquerret Can I rephrase this to go for 10.6 if you're on 12.2 or higher ?

**Reply by gquerret** [reply, marked as answer]
You can upgrade to 10.6, but you can't use the same SonarLint version on 12.2 and 12.8 (see [here](https://github.com/Riverside-Software/sonar-openedge/wiki/SonarLint-for-Eclipse#download-and-install)).
I think the 10.x will become LTA quite soon, 10.6 is stable (using this version internally, and a few customers already moved to 10.x).
